test/lit.cfg

```diff
@@ -1942,3 +1942,9 @@ if copy_env is not None:
     config.environment[key] = os.environ[key]
 
 lit_config.note("Available features: " + ", ".join(sorted(config.available_features)))
+
+# On macOS reflection information is read through the dyld APIs
+# On other platorms, this information is exported through extra
+# entry points in the Swift runtime that are only available in debug builds
+if platform.system() == 'Darwin' or lit_config.debug:
```

Comment:
Rather than adding an additional feature, I would just check for asserts in the test itself.